10/README.md

```diff
@@ -0,0 +1,88 @@
+```
+shortname: 10/CBD
+Name: Consolidating BigchainDB 2.0 Design
+type: Informational
+status: Raw
+Editors: Alberto Granzotto <alberto@bigchaindb.com>
+Contributors: Vanshdeep Singh <vanshdeep@bigchaindb.com>
+```
+
+# Consolidating BigchainDB 2.0 Design
+
+## Abstract
+This document describes an experimental architecture for BigchainDB Server, illustrated in the diagram below. It replaces a small amount of Python code with Go code, allowing the BigchainDB team to experiment with Golang with low risk while achieving several other goals, including easier migration to new versions of Tendermint and higher performance.
+
+## Motivation
+BigchainDB 2.0 Alpha 1 has been developed using BigchainDB 1.3 as a starting point. Design decisions that were correct for the first generation of BigchainDB, are now challenged by the integration of a new consensus algorithm (managed by Tendermint). Specifically, the Tendermint state machine runs in a separate process, and communicates with BigchainDB using multiple channels.
+
+BigchainDB 2.0 Alpha 1 is now the composition of three different system (MongoDB, Tendermint, and BigchainDB itself). This setup generates different problems: a) managing multiple services together adds overhead and complicates deployment; b) the current design makes some calls going from BigchainDB to Tendermint to BigchainDB again, penalizing performance; c) there is data duplication between LevelDB and MongoDB; d) we need to maintain [`py-abci`][pyabci], while the official implementation of the [ABCI][abci] interface is in Golang; e) the logic to validate transactions is spread across multiple files and it's difficult to manage, and a heavy refactoring is needed; f) the configuration is spread across BigchainDB and Tendermint.
+
+In this BEP, we discuss a possible path towards an architecture more suitable for BigchainDB 2.0.
```

Comment:
This paragraph, and those that follow, aren't really about the "Motivation" for the BEP any more. Maybe you could insert a new section header here, such as "Overview".